.pre-commit-config.yaml

```
@@ -23,4 +25,11 @@ repos:
     rev: 6.0.0
     hooks:
-    - id: flake8
\ No newline at end of file
+    - id: flake8
+
+# -   repo: https://github.com/tconbeer/sqlfmt
+#     rev: v0.23.2
+#     hooks:
+#     -   id: sqlfmt
+#         language_version: python
+#         additional_dependencies: ['.[jinjafmt]']
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)